client/src/index.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import App from './components/App.jsx'
+import Login from './components/auth/Login.jsx'
+import AuthLogin from './components/auth/AuthLogin.js'
+import AuthRegister from './components/auth/AuthRegister.js'
+import AuthResetPassword from './components/auth/AuthResetPassword.js'
+import AuthPasswordReset from './components/auth/AuthPasswordReset.js'
+import PasswordReset from './components/auth/PasswordReset.jsx'
+import Register from './components/auth/Register.jsx'
+import { Router, Route,IndexRoute,browserHistory } from 'react-router';
+import { requireAuth,noRequireAuth } from './components/utils/AuthService';
+import {Provider} from 'react-redux'
```

Comment:
imports from packages should always come before relative imports. Please try and reorder the imports in all files.